package.json

```diff
@@ -392,6 +392,14 @@
     "workspaceContains:**/*.cake"
   ],
   "contributes": {
+    "breakpoints": [
```

Comment:
Was this not already here?

Reply:
We are still on a deprecated API for breakpoints https://github.com/OmniSharp/omnisharp-vscode/issues/3512

Reply:
It's unclear to me, should I not have copied these over? It seemed like they got added by @captainsafia's work. If they SHOULDN'T be copied to here I suggest we note that in the AspNetCore-Tooling repo's package.json to avoid repeats.

Reply:
I opened https://github.com/OmniSharp/omnisharp-vscode/pull/3912 to move us off the deprecated API

Reply:
And merged. We now contribute breakpoints.